source/adiabatic_conditions/initial_profile.cc

```diff
@@ -284,6 +306,18 @@ namespace aspect
                              "profile. The higher the number of points, the more accurate "
                              "the downward integration from the adiabatic surface "
                              "temperature will be.");
+          prm.declare_entry ("Use surface condition function", "false",
+                             Patterns::Bool(),
+                             "Whether to use the surface_conditions_function to determine surface "
+                             "conditions, or the adiabatic_surface_temperature and surface_pressure "
+                             "parameters. If this is set to true the reference profile is updated "
+                             "every timestep.");
```

Comment:
can you document here or somehow in the function below what the arguments and outputs of that function are?